src/components/Button/Button.scss

```
@@ -196,4 +196,14 @@ $black-outline-hover: rgba(0, 0, 0, 0.08);
             }
         }
     }
+
+    &.deriv-button__disable-hover {
```

```suggestion
    &.deriv-button__hover--disabled {
```
to follow BEM . also you can move it to line 26 ( before size properties )